vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -195,10 +195,7 @@ std::string GenerateVerifyLink(ledger::ExternalWalletPtr wallet) {
   }
 
   switch (wallet->status) {
-    case ledger::WalletStatus::PENDING: {
-      url = GetSecondStepVerify();
-      break;
-    }
+    case ledger::WalletStatus::PENDING:
```

Comment:
not related to the fix, just a small optimization so that we can easily see what link will be generated